tensorboard/uploader/proto/export_service.proto

```diff
@@ -15,6 +16,8 @@ service TensorBoardExporterService {
   // Stream scalars for all the runs and tags in an experiment.
   rpc StreamExperimentData(StreamExperimentDataRequest)
       returns (stream StreamExperimentDataResponse) {}
+  // Stream BLOB as chunks for a given blob_id.
+  rpc StreamBlobData(StreamBlobRequest) returns (stream StreamBlobResponse) {}
```

Comment:
Request/response type names should match method name: add `Data`.

Reply:
Done.